app/models/miq_event_handler/runner.rb

```diff
@@ -1,2 +1,9 @@
 class MiqEventHandler::Runner < MiqQueueWorkerBase::Runner
+  def self.miq_messaging_service
+    "manageiq.ems-events"
```

Comment:
TODO: we might be able to build this from the existing queue_name for the worker (e.g. `"manageiq.#{queue_name}"`)